src/main/java/com/compilercharisma/chameleonbusinessstudio/controller/AuthenticationController.java

```diff
@@ -1,49 +1,14 @@
 package com.compilercharisma.chameleonbusinessstudio.controller;
 
-import com.compilercharisma.chameleonbusinessstudio.service.AuthenticationService;
-import com.compilercharisma.chameleonbusinessstudio.exception.NoUserLoggedInException;
-import com.compilercharisma.chameleonbusinessstudio.exception.UserNotRegisteredException;
-import com.compilercharisma.chameleonbusinessstudio.entity.user.AbstractUser;
-import java.util.Map;
-import org.springframework.beans.factory.annotation.Autowired;
-import org.springframework.security.core.annotation.AuthenticationPrincipal;
-import org.springframework.security.oauth2.core.user.OAuth2User;
-import org.springframework.stereotype.Controller;
+import org.springframework.security.core.Authentication;
+import org.springframework.security.oauth2.client.authentication.OAuth2AuthenticationToken;
 import org.springframework.web.bind.annotation.*;
 
-/**
- * this class is responsible for handling the "login with Google" aspect of the
- * project.
- * 
- * @author Matt Crow <mattcrow19@gmail.com>
- */
-@Controller
-@RequestMapping("auth/userinfo.email")
-public class AuthenticationController {    
-    private final AuthenticationService authService;
```

Comment:
Given that we're no longer using our own Auth service here, will this controller still support all functionality we had with our own auth service? I see `AuthenticationService.java` provides methods for `getLoggedInUser()` , `isUserRegistered()`, etc and we can accomplish all of that using just the `Authentication` object?

Reply:
I think this would be easier to explain in a call lol